homeassistant/components/peco/strings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "PECO Outage Counter",
```

Comment:
Should this be translated? How would one translate PECO?

Additionally, by default, we use the integration name from the manifest, so this is duplicate. Please remove it.